### PR TITLE
Post and get data using api

### PR DESCRIPTION
**Pull request by Obimbo07**
This pull request introduces the following:

- data constructor
- addscore.js scripts to post data
- getscore.js scripts to display data
- the refresh button listener to display data

### CODE REVIEW

File: src/modules/addscore.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  try {
+    const response = await fetch(scoreURL, {
+      method: 'POST',
+      headers: {
+        'content-Type': 'application/json',
+      },
+      body: requestBody,
+    });
+
```
Comment (Reem-lab) [inline src/modules/addscore.js:13]: - [ ] Great job working on the functionality so far but I think here when I try to press on Refresh button I can't see the scores displayed on the board so kindly check where the problem is. And try to solve it to let the scores displayed on the board after pressing on Refresh button ✨💯🚀✅🤩






https://github.com/Obimbo07/leaderboard-API/assets/58553711/cb8a4c3f-2070-493e-ac2a-8bb6b596d4ec

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Obimbo07, 👋🏻👋🏻

Great job so far! ✨⭐🤩✅
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
- [ ] No Linters error ✅
- [ ] Followed Gitflow ⭐
- [ ] Great job working on the functionality of the API✨
- [ ] Descriptive PR 🚀

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/addscore.js (hunk 1 above)
Comment (Whoistolu) [reply src/modules/addscore.js:13]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Obimbo07 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**